styles.css

```diff
@@ -93,3 +94,142 @@ header {
   padding: 5px;
   column-gap: 15px;
 }
+
+.works {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  font-family: 'Poppins', sans-serif;
+  background-color: white;
+}
+
+.project {
+  margin: 10px;
+  padding: 20px;
+  border-radius: 4px;
+  text-align: left;
+}
+
+.tonic {
+  font-family: 'Poppins', sans-serif;
+  font-size: 32px;
+  font-weight: 700;
+  line-height: 44px;
+  text-align: left;
+  color: #172b4d;
+}
+
+.canopy {
+  font-family: 'Poppins', sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  text-align: left;
+  color: #344563;
+}
+
+.cano {
+  font-family: 'Poppins', sans-serif;
+  font-size: 13px;
+  font-weight: 600;
+  line-height: 16px;
+  text-align: left;
+  color: #7a869a;
+  display: inline-flex;
+  gap: 30px;
+  align-items: center;
+}
+
+.list {
+  display: flex;
+}
+
+.button {
+  font-family: 'Poppins', sans-serif;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  text-align: center;
+  width: 150px;
+  color: #396df2;
+  background-color: white;
+  border: #396df2 solid 2px;
+  border-radius: 10px;
+  height: 50px;
+}
+
+.button:hover {
+  color: #c16e3a;
+}
+
+.p {
+  font-family: 'Poppins', sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+  line-height: 24px;
+  text-align: left;
+  color: #344563;
+}
+
+.lang {
+  display: flex;
+  list-style-type: none;
+  text-decoration: none;
+  margin: 0;
+  padding: 0;
+  margin-right: 10px;
+  gap: 10px;
+}
+
+.lang li {
+  background-color: #ebebff;
+  border-radius: 10px;
+  color: #6070ff;
+  padding: 10px;
+  margin-bottom: 20px;
+}
+
+.about {
+  display: flex;
+  flex-direction: column;
+  background-image: url(images/Base\ about\ my\ self.png);
+  margin: 10px;
+}
+
+.myself {
+  color: #172b4d;
+  font-family: 'Poppins', sans-serif;
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: left;
+  padding-top: 10px;
+}
+
+.AboutMyself {
+  font-family: 'Poppins', sans-serif;
+  font-size: 18px;
+  font-weight: 580;
+  line-height: 24px;
+  text-align: left;
+  color: #344563;
+  padding-bottom: 10px;
+}
+
+.langu {
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- [ ] Please adhere to the design as much as possible because the more you try to make it similar to the `Figma` design the more it will look professional. Please use the correct `font-family`, `font-weight`, `colour` and spacing in your this section child content. Kindly take a look at the below image and try to fix the UI issues.

<table>
<tr>
<th>Figma Design</th>
<th>Your Design</th>
</tr>
<tr>
<td>

![Screenshot from 2023-04-28 17-17-40](https://user-images.githubusercontent.com/66476329/235146184-bedc1439-529c-49ef-9479-20df25bcc654.png)

</td>
<td>

![Screenshot from 2023-04-28 17-18-35](https://user-images.githubusercontent.com/66476329/235146216-b39f3593-fbdb-4bb9-873f-5879ddecbaf9.png)

</td>
</tr>
</table>